gramlib/grammar.ml

```diff
@@ -720,7 +720,7 @@ and print_level : type s. _ -> _ -> s ex_symbols list -> _ =
       (fun sep (ExS rule) ->
          fprintf ppf "%t%a" sep print_rule rule;
          fun ppf -> fprintf ppf "%a| " pp_print_space ())
-      (fun ppf -> ()) rules
+      (fun ppf -> ()) rules ppf
```

Comment:
I guess 08 detects the partial application? Why did this even work before?

Reply:
It did not.